Jenkinsfile

```diff
@@ -32,7 +32,6 @@ node('ios-osx') {
         // load/restore carthage build directory
         sh '''#!/bin/bash -l
             set -x
-            set -e
```

Comment:
would prefer to fail on error.

on this occasion please rewrite this block, to be multiline and set env variable with `withEnv` pipeline helper